extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -94,14 +94,14 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.requests_ = {};
 
     /**
-     * @private {JSONType}
+     * @private {JsonObject}
      */
-    this.config_ = /** @type {JSONType} */ ({});
+    this.config_ = /** @type {JsonObject} */ ({});
```

Comment:
would it be better to call `dict()` here? (besides the implication of adding additional code to the binary)